### PR TITLE
Bug fix: Bootstrap configuration for tjdashboard

### PR DESCRIPTION
**Pull request by shitalkakade24**

### CODE REVIEW

File: src/components/com_tjdashboard/site/views/dashboard/tmpl/default.php [hunk 1]
```diff
@@ -29,11 +29,14 @@
...
-
-<div class="row-fluid">
-	<div class="col-xs-12 tjdashboard" style="margin-left:10px;">
-		<h3>
-			<div data-dashboard-id="<?php echo $this->item->dashboard_id;?>" class="tjdashboard-title"><?php echo htmlspecialchars($this->item->title);?></div>
-		</h3>
+<div class="<?php echo COM_TJDASHBOARD_WRAPPER_DIV; ?> tjBs3">
+	<div class="container-fluid">
+		<div class="row">
+			<div class="col-xs-12 tjdashboard" style="margin-left:10px;">
```
Comment (vaivk369) [inline src/components/com_tjdashboard/site/views/dashboard/tmpl/default.php]: @shitalkakade24  remove inline css. Check if you need this styling if yes add in css file

Comment (shitalkakade24) [reply src/components/com_tjdashboard/site/views/dashboard/tmpl/default.php]: removed. no need.

**Comment by vaivk369** [comment]
@manojLondhe this looks ok to me. Please check
cc @coolbung

File: src/components/com_tjdashboard/administrator/languages/en-GB/en-GB.com_tjdashboard.ini [hunk 2]
```diff
@@ -1,6 +1,11 @@
 COM_TJDASHBOARD="TjDashboard"
 COM_TJDASHBOARD_COMPONENT_LABEL="Tjdashboard"
 COM_TJDASHBOARD_CONFIGURATION="Tjdashboard Configuration"
+
+; bootstrap configue
```
Comment (manojLondhe) [inline src/components/com_tjdashboard/administrator/languages/en-GB/en-GB.com_tjdashboard.ini]: `; bootstrap configue` => `; Bootstrap configuration`

File: src/components/com_tjdashboard/administrator/languages/en-GB/en-GB.com_tjdashboard.ini [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
+COM_TJDASHBOARD_LOAD_BOOTSTRAP="Load bootstrap manually"
+COM_TJDASHBOARD_LOAD_BOOTSTRAP_DESC="If set to 'Yes', bootstrap css will be loaded from TjDashboard component"
```
Comment (manojLondhe) [inline src/components/com_tjdashboard/administrator/languages/en-GB/en-GB.com_tjdashboard.ini]: CSS

File: src/components/com_tjdashboard/site/tjdashboard.php [hunk 4]
```diff
@@ -18,6 +18,26 @@
...
+{
+	// Load bootstrap CSS and JS.
+	JHtml::stylesheet('media/techjoomla_strapper/bs3/css/bootstrap.css');
+
+	JHtml::_('bootstrap.framework');
+}
+
+if (JVERSION < '3.0')
+{
+	define('COM_TJDASHBOARD_WRAPPER_DIV', 'techjoomla-bootstrap tjlms-wrapper');
```
Comment (manojLondhe) [inline src/components/com_tjdashboard/site/tjdashboard.php]: what is `tjlms` used in `tjdashboard` for?

File: src/components/com_tjdashboard/site/tjdashboard.php [hunk 5]
```diff
@@ -18,6 +18,26 @@
...
+
+if ($tjdashboardparams->get('load_bootstrap') == 1)
+{
+	// Load bootstrap CSS and JS.
+	JHtml::stylesheet('media/techjoomla_strapper/bs3/css/bootstrap.css');
+
+	JHtml::_('bootstrap.framework');
+}
+
+if (JVERSION < '3.0')
```
Comment (manojLondhe) [inline src/components/com_tjdashboard/site/tjdashboard.php]: if we are not supporting j2.5.x, we don't need `JVERSION` check

File: src/components/com_tjdashboard/site/tjdashboard.php [hunk 6]
```diff
@@ -18,6 +18,26 @@
...
+{
+	define('COM_TJDASHBOARD_WRAPPER_DIV', 'techjoomla-bootstrap tjlms-wrapper');
+}
+else
+{
+	define('COM_TJDASHBOARD_WRAPPER_DIV', 'tjlms-wrapper');
```
Comment (manojLondhe) [inline src/components/com_tjdashboard/site/tjdashboard.php]: don't use  `tjlms`

File: src/components/com_tjdashboard/site/views/dashboard/tmpl/default.php [hunk 7]
```diff
@@ -29,11 +29,14 @@
 		TJDashboardUI.initDashboard(<?php echo ($this->item->state == 1? $this->item->dashboard_id : 0); ?>);
 	});
 </script>
-
-<div class="row-fluid">
-	<div class="col-xs-12 tjdashboard" style="margin-left:10px;">
-		<h3>
-			<div data-dashboard-id="<?php echo $this->item->dashboard_id;?>" class="tjdashboard-title"><?php echo htmlspecialchars($this->item->title);?></div>
-		</h3>
+<div class="<?php echo COM_TJDASHBOARD_WRAPPER_DIV; ?> tjBs3">
```
Comment (manojLondhe) [inline src/components/com_tjdashboard/site/views/dashboard/tmpl/default.php]: If you are loading a constant for `COM_TJDASHBOARD_WRAPPER_DIV`, make `tjBs3` as part of wrapper class
eg. `COM_TJDASHBOARD_WRAPPER_DIV="class1 class2 tjBs3"`

File: src/components/com_tjdashboard/site/views/dashboard/tmpl/default.php [hunk 8]
```diff
@@ -29,11 +29,14 @@
 		TJDashboardUI.initDashboard(<?php echo ($this->item->state == 1? $this->item->dashboard_id : 0); ?>);
 	});
 </script>
```
Comment (manojLondhe) [inline src/components/com_tjdashboard/site/views/dashboard/tmpl/default.php]: Give a blank line after closing `</script>` tag

**Review by manojLondhe — CHANGES_REQUESTED** [review]
@shitalkakade24 do look at suggested changes
cc @twsvaishali

**Comment by manojLondhe** [comment]
@shitalkakade24 also follow commit message format.

File: src/components/com_tjdashboard/administrator/languages/en-GB/en-GB.com_tjdashboard.ini (hunk 2 above)
Comment (shitalkakade24) [reply src/components/com_tjdashboard/administrator/languages/en-GB/en-GB.com_tjdashboard.ini]: changed

File: src/components/com_tjdashboard/administrator/languages/en-GB/en-GB.com_tjdashboard.ini (hunk 3 above)
Comment (shitalkakade24) [reply src/components/com_tjdashboard/administrator/languages/en-GB/en-GB.com_tjdashboard.ini]: changed

File: src/components/com_tjdashboard/site/tjdashboard.php (hunk 4 above)
Comment (shitalkakade24) [reply src/components/com_tjdashboard/site/tjdashboard.php]: removed

File: src/components/com_tjdashboard/site/tjdashboard.php (hunk 5 above)
Comment (shitalkakade24) [reply src/components/com_tjdashboard/site/tjdashboard.php]: removed

File: src/components/com_tjdashboard/site/tjdashboard.php (hunk 6 above)
Comment (shitalkakade24) [reply src/components/com_tjdashboard/site/tjdashboard.php]: remove

File: src/components/com_tjdashboard/site/views/dashboard/tmpl/default.php (hunk 7 above)
Comment (shitalkakade24) [reply src/components/com_tjdashboard/site/views/dashboard/tmpl/default.php]: changed

File: src/components/com_tjdashboard/site/views/dashboard/tmpl/default.php (hunk 8 above)
Comment (shitalkakade24) [reply src/components/com_tjdashboard/site/views/dashboard/tmpl/default.php]: done

**Comment by manojLondhe** [comment]
@shitalkakade24 @twsvaishali @thite-amol Does this PR assume TJ Strapper is installed?

**Pull request merged**